packages/aragon-wrapper/src/rpc/handlers/past-events.js

```diff
@@ -0,0 +1,14 @@
+export default function (request, proxy, wrapper) {
+  const eventsOptions = {}
+  const [fromBlock, toBlock] = request.params
+
+  if (fromBlock) {
+    eventsOptions.fromBlock = fromBlock
+  }
+
+  if (toBlock) {
+    eventsOptions.toBlock = toBlock
+  }
+
+  return proxy.pastEvents(null, eventsOptions)
+}
```

Comment:
What do you think about abstracting this all into the `Proxy.events()`, so any user of the Proxy will get these benefits? We can emit a special sentinel "event" to notify the user that they've gone from `eth_getlogs` to `eth_subscribe`.

Reply:
Discussed this offline and decided to keep the split between the `pastEvents()` and `events()`:

- More similar to native web3.js API (`getPastEvents()` and `events()`)
- Including this inside the `@aragon/wrapper`'s `Proxy` is not immediately useful for that package
- Scopes these changes to aragonAPI, where it's most useful